website/data/docs-nav-data.json

```
@@ -809,6 +814,10 @@
       }
     ]
   },
+  {
+    "title": "External →",
```

Loosely held hot take, but I think we should drop the arrow as it's not something we've really designed for in the nav. Would also suggest we could change the text to `External plugins` just for extra clarity:

```suggestion
    "title": "External Plugins",
```

> ☝️ Would love Packer team input on this as well, in case that's not obvious!